components/ContentSearch/index.js

```diff
@@ -25,14 +25,22 @@ const ContentSearch = ({ onSelectItem, placeholder, label, contentTypes, mode, e
 	 *
 	 * @param {*} item
 	 */
-	function handleSelection(item) {
+	function handleOnNavigate(item) {
 		if (item === 0) {
 			setSelectedItem(null);
 		}
 
 		setSelectedItem(item);
 	}
 
+	/**
+	 * handleItemSelection
+	 *
+	 * reset the search input & item container
+	 * trigger the onSelectItem callback passed in via props
+	 *
+	 * @param {*} item
+	 */
 	function handleItemSelection(item) {
```

Comment:
I think this would make the function names even more consistent. What do you think?

```suggestion
	function handleOnItemSelect(item) {
```